### ISSUE TITLE
Remove lives

### ISSUE THREAD

**Issue by nvs** [nvs/gem#435]
Lives suck. They promote a risky style of play where you can sort of sometimes recover because, hey, you won't fully leak from a single monster when you have `15` lives until Level 42 and beyond. They cause glaring mistakes in strategy to not be as heavily punished. You probably should leak and lose, but instead you can limp along further. And they throw new players who assume that there will be a one-to-one association between monsters and lives lost. And a lot of players feel that leaking at all should mean you lose.

About the only good thing about lives is that it provides a safety net for new players. And, the notion that the lives lost equals the gold reward is interesting (to me). But not interesting enough to make me like lives.

So, the idea is to make all leaks before a certain point not cause someone to lose. But, after such point any leak will equate to a loss. This is Gem. We don't hold hands (much). Effectively, this would make the period until that threshold a sort of extended tutorial. New players can limp along and get a feel for things, maybe make a combination or two. Then get brutally destroyed by say, a boss. As far as leaking goes, you lose time and gold. And time is de facto goal in Gem. No balance issues here. So, for anyone who can get past the boss, the training wheels come on. You leak, you lose. The end.

But, this is blocked by #436. That boss is the 'certain point' previously mentioned. If you clear the boss, you can then proceed to the rest of the game.

**Comment by nvs** [comment]
As #438 is being put on hold, we can tackle this one now. No boss.

TL;DR:
- No lives at all.
- Leaks on and before Level 9 only cost the time and gold of the leak itself.
- Any leak on Level 10 and beyond equates to a loss.
